### ISSUE TITLE
Is this a ghost Repo??!!

### ISSUE THREAD

**Issue by FarshadN** [okonek/tidal-cli-client#80]
Over 10 registered issues with over 10 comments each from users about failed sign on, and no response. Is Tidal even aware of this issue?!

**Comment by Lyndeno** [comment]
This isn't an official Tidal application.

**Comment by josh231101** [comment]
Is this safe? Like for real, can I access to my account safe?

**Comment by aleksanderjessitm** [comment]
Yeah, this is a repo that was last updated 3 years ago. I'd say there is a high chance that this project is indeed abandoned.

**Comment by aleksanderjessitm** [comment]
The solution [in this repo](https://github.com/lucaslg26/TidalAPI) might still apply. Though you'd still need a Windows or a macOS machine, I believe, **unless** you get the token from a request made by the browser app.

**Comment by aleksanderjessitm** [comment]
> Is this safe? Like for real, can I access to my account safe?

Any app may be unsafe. It's your choice whether to trust the author or not.
